src/libstd/net/tcp.rs

```diff
@@ -129,6 +132,23 @@ impl TcpStream {
     ///     println!("Couldn't connect to server...");
     /// }
     /// ```
+    ///
+    /// Open a TCP connection to `127.0.0.1:8080`. If the connection fails, open
+    /// a TCP connection to `127.0.0.1:8081`:
+    ///
+    /// ```no_run
+    /// use std::net::TcpStream;
```

Comment:
The doc tests failed, `SocketAddr` needs to be imported first.

<details>

```
[00:57:39] failures:
[00:57:39] 
[00:57:39] ---- net/tcp.rs - net::tcp::TcpListener::bind (line 602) stdout ----
[00:57:39] 	error[E0433]: failed to resolve. Use of undeclared type or module `SocketAddr`
[00:57:39]  --> net/tcp.rs:7:5
[00:57:39]   |
[00:57:39] 7 |     SocketAddr::from(([127, 0, 0, 1], 80)),
[00:57:39]   |     ^^^^^^^^^^ Use of undeclared type or module `SocketAddr`
[00:57:39] 
[00:57:39] error[E0433]: failed to resolve. Use of undeclared type or module `SocketAddr`
[00:57:39]  --> net/tcp.rs:8:5
[00:57:39]   |
[00:57:39] 8 |     SocketAddr::from(([127, 0, 0, 1], 443)),
[00:57:39]   |     ^^^^^^^^^^ Use of undeclared type or module `SocketAddr`
[00:57:39] 
[00:57:39] thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:280:12
[00:57:39] note: Run with `RUST_BACKTRACE=1` for a backtrace.
[00:57:39] 
[00:57:39] ---- net/tcp.rs - net::tcp::TcpStream::connect (line 140) stdout ----
[00:57:39] 	error[E0433]: failed to resolve. Use of undeclared type or module `SocketAddr`
[00:57:39]  --> net/tcp.rs:7:5
[00:57:39]   |
[00:57:39] 7 |     SocketAddr::from(([127, 0, 0, 1], 8080)),
[00:57:39]   |     ^^^^^^^^^^ Use of undeclared type or module `SocketAddr`
[00:57:39] 
[00:57:39] error[E0433]: failed to resolve. Use of undeclared type or module `SocketAddr`
[00:57:39]  --> net/tcp.rs:8:5
[00:57:39]   |
[00:57:39] 8 |     SocketAddr::from(([127, 0, 0, 1], 8081)),
[00:57:39]   |     ^^^^^^^^^^ Use of undeclared type or module `SocketAddr`
[00:57:39] 
[00:57:39] thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:280:12
[00:57:39] 
[00:57:39] 
[00:57:39] failures:
[00:57:39]     net/tcp.rs - net::tcp::TcpListener::bind (line 602)
[00:57:39]     net/tcp.rs - net::tcp::TcpStream::connect (line 140)
```

</details>